roachtest: fix repro command

tbg opened the pull request:
The link wasn't rendering correctly. This is now fixed and tested.

Release note: None

tbg committed:
roachtest: fix repro command

The link wasn't rendering correctly. This is now fixed and tested.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/68223)
<!-- Reviewable:end -->

erikgrinaker commented on pkg/cmd/internal/issues/testdata/post/failure-with-url-matching-and-related-issue.txt:
````diff
@@ -0,0 +1,39 @@
+boom
+```
+<details><summary>Reproduce</summary>
+<p>
+
+
+See: [FooBar README](https://github.com/cockroachdb/cockroach)
+
+<p>Parameters in this failure:
````
These nested `<p>` tags seem off. In HTML, the second `<p>` will implicitly close the previous one, and so we now have additional spurious `</p>` tags below.

tbg replied:
Good point, fixed in a new commit.

tbg committed:
issues: produce valid(er) html

We were nesting `<p>` tags before. Now we don't. The rendering appears
to not have changed as a result in the few examples I spot checked.

Release note: None

tbg commented:
Hmm, can't push right now, so the commit isn't there yet. Just a heads up.

tbg commented:
Now it's there.

erikgrinaker reviewed (approved):
Looks good, thanks!

tbg commented:
bors r=erikgrinaker
